ImportError: cannot import name '_endpoint_from_view_func'

```
Traceback:
/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/_pytest/python.py:578: in _importtestmodule
    mod = import_path(self.fspath, mode=importmode)
/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/_pytest/pathlib.py:524: in import_path
    importlib.import_module(module_name)
/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1006: in _gcd_import
    ???
<frozen importlib._bootstrap>:983: in _find_and_load
    ???
<frozen importlib._bootstrap>:967: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:677: in _load_unlocked
    ???
/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/_pytest/assertion/rewrite.py:170: in exec_module
    exec(co, module.__dict__)
test_symbol_info.py:2: in <module>
    from package.symbol_info import get_udf_data
__init__.py:5: in <module>
    from package.chart_config import Config
chart_config.py:3: in <module>
    from flask_restful import Resource
/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/flask_restful/__init__.py:14: in <module>
    from flask.helpers import _endpoint_from_view_func
E   ImportError: cannot import name '_endpoint_from_view_func' from 'flask.helpers' (/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/flask/helpers.py)
```

Environment:

- Python version: 3.7
- Flask version: latest release

--- a/flask_restful/__init__.py
+++ b/flask_restful/__init__.py
@@ -11,7 +11,6 @@ from werkzeug.wrappers import Response as ResponseBase
 from flask_restful.utils import http_status_message, unpack, OrderedDict
 from flask_restful.representations.json import output_json
 import sys
-from flask.helpers import _endpoint_from_view_func
 from types import MethodType
 import operator
 try:
@@ -156,7 +155,7 @@ class Api(object):
             rule = blueprint_setup.url_prefix + rule
         options.setdefault('subdomain', blueprint_setup.subdomain)
         if endpoint is None:
-            endpoint = _endpoint_from_view_func(view_func)
+            endpoint = view_func.__name__
         defaults = blueprint_setup.url_defaults
         if 'defaults' in options:
             defaults = dict(defaults, **options.pop('defaults'))

